packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -279,7 +279,7 @@ export class GCTelemetryTracker {
 				const event = {
 					eventName: `${state}Object_${usageType}`,
 					...tagCodeArtifacts({ pkg: packagePath?.join("/") }),
-					stack: generateStack(),
+					stack: generateStack(30),
```

Comment:
Do you look at InactiveObject_Loaded?  I would think Tombstones would be all you need/want to update.  If these are still (relatively) rare in telemetry, no harm in doing this though.

Reply:
Yes, we do look at InactiveObject_Loaded still. I think the events are relatively rare, though we saw a spike just yesterday. So far, it seems to have returned to more normal levels of telemetry today, so I'll opt to keep this change in.